src/ucp/wireup/wireup_cm.c

```diff
@@ -330,6 +331,12 @@ static ucs_status_t ucp_cm_ep_init_lanes(ucp_ep_h ep,
     return status;
 }
 
+static unsigned ucp_cm_pack_flags(ucp_worker_h worker)
```

Comment:
maybe `ucp_worker_cm_pack_flags`?

Reply:
most of the funcs in this file are called "ucp_cm_.."

Reply:
`ucp_cm_worker_address_pack_flags` or shorter `ucp_cm_address_pack_flags`?

Reply:
`ucp_cm_address_pack_flags` - sounds good